index.src.html

```diff
@@ -2344,13 +2362,38 @@
             |clone|.
     2. Return |clone|.
 
+: To check if a {{AudioDataInit}} is a
+    <dfn>valid AudioDataInit</dfn>, run these steps:
+:: 1. If {{AudioDataInit/sampleRate}} = 0, return `false`.
+    2. If {{AudioDataInit/numberOfFrames}} = 0, return `false`.
+    3. If {{AudioDataInit/numberOfChannels}} = 0, return `false`.
+    4. Return `true`.
+
+<div class='note'>
+Note: An {{AudioDataInit}}'s {{AudioDataInit/data}} must have the following
```

Comment:
This might not be the right place for this. AudioDataInit's members do not have any definitions. Should they?

Reply:
A dict is generally defined by how it's used when it's the input other objects like this, but a sentence explaining what it is wouldn't hurt I guess.

Reply:
I lean toward being more zen here, letting the members be implicitly defined by their usage.

If there's anything in this note that isn't also in the validation steps, I'd definitely add it there.

Reply:
Ok. I've removed most of the note. It's now just a small comment, which serves as a pointer to the [=planar=] and [=interleaved=] formats.